package.json

```diff
@@ -137,6 +137,7 @@
     "postcss-preset-env": "6.5.0",
     "postcss-safe-parser": "^4.0.1",
     "postcss-scss": "^1.0.2",
+    "prettier": "^2.1.2",
```

Comment:
why did this get added here? its not used

Reply:
It is required for inline snapshots